sycl/test-e2e/NewOffloadDriver/buffer.cpp

```diff
@@ -0,0 +1,48 @@
+// REQUIRES: level_zero
+// RUN: %{build} --offload-new-driver -o %t.out
```

Comment:
to force JIT i think we can do
`%clangxx -fsycl --offload-new-driver ...`
because if it's set up for AOT, `%{build}` will expand to AOT options